ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.ts

```diff
@@ -1465,179 +1470,37 @@ export class ParticipantListComponent implements OnInit {
   }
 
   public isSortField(name: string): boolean {
-    return name === this.sortField;
+    if (this.sortBy) {
+      return name === this.sortBy.innerProperty;
+    } else {
+      return false;
+    }
```

Comment:
Lines 1473-1477 can be shortened 
like
`return this.sortBy && name === this.sortBy.innerProperty`

or even 
`return name === this.sortBy?.innerProperty`